### DISCUSSION TITLE
Is it a bug?

### DISCUSSION THREAD

**Discussion by DmitryHudrich** [Orange-OpenSource/hurl#3094] [Q&A] [answered]
I am doing `hurl main.hurl` where `main.hurl` contains few requests, but only last one of them executes. I use arch linux and hurl from extra repository.
e.g.
```
# Get home:
GET https://pokeapi.co/api/v2/pokemon/ditto

GET https://pokeapi.co/api/v2/ability/battle-armor
```

**Comment by jcamiel** [comment, marked as answer]
Hi @DmitryHudrich, you can see what's going with `--verbose`. Usually when your command is:

```shell
$ hurl main.hurl
```

Every requests of `main.hurl` are executed but only the last response of the file is outputed on standard output.

If you want to see response on every request, you can:

- use `--very-verbose`: each response will be logged on standard error
- use `[Options]` section to output a particular response:

```hurl
# Get home:
GET https://pokeapi.co/api/v2/pokemon/ditto
[Options]
output: -

GET https://pokeapi.co/api/v2/ability/battle-armor
```

`output: -` will print the request response on standard ouput
`output: myfile.bin` will save the request response on the file `myfile.bin`

**Comment by DmitryHudrich** [comment]
thanks for you answer!
